src/main.c

```diff
@@ -8,111 +8,146 @@
 #include "../include/floor.h"
 #include "../include/player.h"
 #include "../include/monster_hashtbl.h"
-#include "../include/player.h"
+#include "../include/maze.h"
+
+
+int event_on_tiles(char * tile, State * state, Direction i) {
+  /* return 1 if the player can go on this tile and 0 otherwise */
+  /* operates here events of special tiles */
+  switch (*tile) {
+    case '#':
+      // empty
+      return 0; break;
```

Comment:
return 0 can be removed as it is returned in the end of the function

Reply:
J'étais d'accord, mais en faisant les tests, c'est pour traiter avec le `default` comme un sol normal.